ocs_ci/ocs/ui/base_ui.py

```diff
@@ -207,6 +207,18 @@ def get_element_text(self, locator):
         """
         return self.driver.find_element(by=locator[1], value=locator[0]).text
 
+    def get_elements(self, locator):
+        """
+        Get an elements list. Useful to count number of elements presented on page, etc.
+
+        Args:
+            locator (set): (GUI element needs to operate on (str), type (By)).
+
+        Return:
+            list: The list of WebElements
+        """
+        return self.driver.find_elements(by=locator[1], value=locator[0])
```

Comment:
Can you pls fetch text of matching elements found and print as Info?

Reply:
If I understood correctly, that you want to print text of every elements, there are few reasons why not to do that: 
1) The elements not always have a text, e.g. navigation btns, icons, infographics, meaning you will see lots of empty lines
2) the rest general functions in base_ui.py such as get_element_text(), check_element_text() don't have print of the text, it will over-inflate logs and not so useful in case when we don't need the text
3) this function is intended to see a number of elements on the page, make assertion e.g.  len(webElementsList) == 0; iterate the list of elements, take a random element from the list, etc. If this will be a crucial logic of the test is
4) it may be so the list of elements has dozens of elements.. too much for printing, and in the same time you always can print their text implicitly, while iterating the list from the test/procedure

Reply:
> If I understood correctly, that you want to print text of every elements, there are few reasons why not to do that:
> 
> 1. The elements not always have a text, e.g. navigation btns, icons, infographics, meaning you will see lots of empty lines
> 2. the rest general functions in base_ui.py such as get_element_text(), check_element_text() don't have print of the text, it will over-inflate logs and not so useful in case when we don't need the text
> 3. this function is intended to see a number of elements on the page, make assertion e.g.  len(webElementsList) == 0; iterate the list of elements, take a random element from the list, etc. If this will be a crucial logic of the test is
> 4. it may be so the list of elements has dozens of elements.. too much for printing, and in the same time you always can print their text implicitly, while iterating the list from the test/procedure

Then why not use the existing function from base_ui?
`def check_element_text(self, expected_text, element="*"):`

Reply:
because it is different:
self.driver.find_element != self.driver.find_elements

Reply:
> because it is different: self.driver.find_element != self.driver.find_elements

Pls check the function again
```
element_list = self.driver.find_elements_by_xpath(
            f"//{element}[contains(text(), '{expected_text}')]"
        )
        return len(element_list) > 0
```

Reply:
please, specify what should I check again?

Reply:
Updated above comment

Reply:
I see this function, this is good function, but not what the test requires. We need to have a common routine to find element list (not verify that the number of elements is larger than 0). The func description is in a docstring.